### PR TITLE
Copyediting of PDD spec

### PR DESCRIPTION
**Pull request by JesseWeinstein**

### CODE REVIEW

**Comment by JesseWeinstein** [comment]
Let me know if you want me to squash these two commits.

File: pdd-protocol-driven-development/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Tests should be deterministic, so that different implementations produce the sam
                 └─────────┘    └───────────────┘
 ```
 
-So that a diff between two results should yield 0 results
+So that a diff between two output files should show not differences.
```
Comment (RichardLitt) [inline pdd-protocol-driven-development/README.md]: Should not show differences?

File: pdd-protocol-driven-development/README.md [hunk 2]
```diff
@@ -7,38 +7,38 @@ Protocol Driven Development
...
-4 - Create the Protocol Compliance Tests (consisting on injecting the packets/messages generated in the last step in the other implementations and comparing outputs)
+1. Define the desired Protocol Spec that is going to be implemented.
+2. Design the compliance tests that prove that a certain implementation conforms with the spec.
+3. Once an implementation is done, capture the messages traded on the wire using that implementation, so that the behaviour of both participants can be replicated without the agent.
+4. Create the Protocol Compliance Tests (consisting of injecting the packets/messages generated in the last step into the other implementations and comparing the outputs).
 
 ## Protocol Spec
 
-Should define the goals, motivation, messages traded between participants and some use cases. It should not cover language or framework specifics.
+A Protocol Spec should define the goals & motivation for the protocol, the messages traded between participants in it, and some use cases. It should not cover language or framework specifics.
```
Comment (RichardLitt) [inline pdd-protocol-driven-development/README.md]: Use an 'and' instead of an ampersand.

**Comment by RichardLitt** [comment]
Two comments, otherwise LGTM.

File: pdd-protocol-driven-development/README.md (hunk 1 above)
Comment (JesseWeinstein) [inline pdd-protocol-driven-development/README.md]: Good catch!

**Comment by RichardLitt** [comment]
Thanks, @JesseWeinstein!

**Pull request merged**